File: Ridy-taxi-app/ios/common/Networking/WebSocket/WriteComplaint.swift
//
//  WriteComplaint.swift
//  Shared
//
//  Created by Manly Man on 11/23/19.
//  Copyright © 2019 Innomalist. All rights reserved.
//

import Foundation

public class WriteComplaint: SocketRequest {
    public typealias ResponseType = EmptyClass
    public var params: [Any]?
    
    public init(requestId: Int, subject: String, content: String) {
        self.params = [
        requestId,
        subject,
        content
        ]
    }
    
    required public init() {}
}

File: Ridy-taxi-app/android/common/src/main/java/com/innomalist/taxi/common/networking/socket/WriteComplaint.kt
package com.innomalist.taxi.common.networking.socket

import com.innomalist.taxi.common.networking.socket.interfaces.SocketRequest

class WriteComplaint(requestId: Long, subject: String, content: String): SocketRequest() {
    init {
        this.params = arrayOf(requestId, subject, content)
    }
}